addons/web/static/src/search/control_panel/control_panel.js

```diff
@@ -40,9 +40,10 @@ const STICKY_CLASS = "o_mobile_sticky";
  */
 
 class EmbeddedActionsConfigHandler {
-    constructor(parentActionId, currentActiveId, ormService) {
+    constructor(parentActionId, currentActiveId, parentResModel, ormService) {
```

Comment:
Changing the arguments order in stable is not recommended. In this case, the class is only used inside that module, so it wouldn't be a big problem though.

Also, changes made in `web` should be supported by a test.

Reply:
Don't worry, I'm ok with the change too. Just wanted to point it out, not block it 😉 .